share/arno-iptables-firewall/environment

```diff
@@ -1614,26 +1614,39 @@ check_interface()
   return 1
 }
 
-# Get IP address/mask of specified network interface
+# Get all IP address(es)/mask(s) of specified network interface
+get_network_ipv4_address_mask_all()
+{
+  ip -o addr show dev "$1" 2>/dev/null \
+    |awk '$3 == "inet" { print $4 }' |tr '\n' ' ' |sed s,' $',,
+}
+
+# Get (primary) IP address/mask of specified network interface
 get_network_ipv4_address_mask()
+{
+  get_network_ipv4_address_mask_all "$1" |cut -d' ' -f1
+}
+
+# Get all IP address(es) of the specified network interface
+get_network_ipv4_address_all()
 {
   ip -o addr show dev "$1" 2>/dev/null \
-    |awk '$3 == "inet" { print $4; exit; }'
+    |awk '$3 == "inet" { print $4 }' |cut -f1 -d'/' |tr '\n' ' ' |sed s,' $',,
```

Comment:
Is the trailing `sed` really needed ?  But if you want to trim trailing spaces `sed 's, *$,,'` would be my choice.

My comment earlier was with regard to not needing `|tr '\n' ' '` as long as the output was parsed using `unset IFS`.  I understand you want to make it more general with spaces.